### ISSUE TITLE
[grpc:x64-windows] build failure

### ISSUE THREAD

**Issue by torstymctorst** [microsoft/vcpkg#9078]
**Host Environment**
- OS: Windows 10 Pro 19.09
- Compiler: VS 2015 x64 

**To Reproduce**
`./vcpkg install grpc`

**Failure logs** 
C:\Werkstatt\vcpkg>vcpkg install grpc
The following packages will be built and installed:
    grpc[core]:x64-windows
Starting package 1/1: grpc:x64-windows
Building package grpc[core]:x64-windows...
-- Note: grpc only supports static library linkage. Building static library.
-- Using cached C:/Werkstatt/vcpkg/downloads/grpc-grpc-v1.23.1.tar.gz
-- Extracting source C:/Werkstatt/vcpkg/downloads/grpc-grpc-v1.23.1.tar.gz
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" -E tar xjf C:/Werkstatt/vcpkg/downloads/grpc-grpc-v1.23.1.tar.gz
    Working Directory: C:/Werkstatt/vcpkg/buildtrees/grpc/src/TEMP
    Error code: 1
    See logs for more information:
      C:\Werkstatt\vcpkg\buildtrees\grpc\extract-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_extract_source_archive.cmake:43 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_extract_source_archive_ex.cmake:108 (vcpkg_extract_source_archive)
  scripts/cmake/vcpkg_from_github.cmake:114 (vcpkg_extract_source_archive_ex)
  ports/grpc/portfile.cmake:13 (vcpkg_from_github)
  scripts/ports.cmake:94 (include)

Error: Building package grpc:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: grpc:x64-windows
  Vcpkg version: 2019.09.12-nohash

 - Failure log:
CMake Error: Problem with archive_write_header(): Can't create ''
CMake Error: Current file: grpc-1.23.1/test/core/http/python_wrapper.sh
CMake Error: Problem extracting tar: C:/Werkstatt/vcpkg/downloads/grpc-grpc-v1.23.1.tar.gz

**Additional context**
`./vcpkg update`
Using local portfile versions. To update the local portfiles, use `git pull`.
No packages need updating.

`git pull`
Already up-to-date.

**Comment by LilyWangL** [comment]
Hi @torstymctorst, I can't reproduce it locally. I installed grpc:x64-windows successfully with vcpkg latest code. I noticed your log has this information:
```
C:/Werkstatt/vcpkg/downloads/grpc-grpc-v1.23.1.tar.gz
Working Directory: C:/Werkstatt/vcpkg/buildtrees/grpc/src/TEMP
```
The directory TEMP should not exist. Could you delete the directory `VCPKG_path/grpc/src` and rebuild it?

**Comment by torstymctorst** [comment]
I deleted the directory C:/Werkstatt/vcpkg/buildtrees/grpc/src.
./vcpkg install grpc still gives the same output.

**Comment by torstymctorst** [comment]
Solved in #8603 . I simply switched off UTF-8 Support in the Windows language settings. Solution from #8553 did not help.

**Comment by andreaszervas** [comment]
> Solved in #8603 . I simply switched off UTF-8 Support in the Windows language settings. Solution from #8553 did not help.
Hi torstymctorst! 
how did you switched off UTF-8 Support? I'm running Windows 7.
Thank you!

**Comment by gebdadendo** [comment]
> Alexej Harm found the setting that causes this [issue](https://gitlab.kitware.com/cmake/cmake/issues/19102#note_558431).
> 
> The UTF-8 beta in intl.cpl > Administrative > Change system locale... needs to be unticked for cmake to function properly.

I did not try this on windows 7 and cannot try now, but on windows 10 this is the solution in the above mentioned context. Sorry for this probably won't help you much :/
